<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memberWindow</class>
 <widget class="QMainWindow" name="memberWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>170</width>
      <height>106</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>106</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color: white;
background-color: rgb(37, 34, 94);</string>
    </property>
    <property name="text">
     <string>Members</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>170</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>60</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>170</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color:white;
background-color: rgb(214, 68, 117);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;GoGym&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; vertical-align:sub;&quot;&gt;Trainee Panel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>170</width>
      <height>106</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>106</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color: white;
background-color: rgb(37, 34, 94);</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>170</width>
      <height>106</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>60</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color: white;
background-color: rgb(37, 34, 94);</string>
    </property>
    <property name="text">
     <string>My Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>170</width>
      <height>106</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>106</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color: white;
background-color: rgb(37, 34, 94);</string>
    </property>
    <property name="text">
     <string>WorkOut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>170</width>
      <height>106</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>106</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 9pt &quot;Segoe UI&quot;;
color: white;
background-color: rgb(37, 34, 94);</string>
    </property>
    <property name="text">
     <string>Attendance Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>170</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(37, 34, 94);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>1110</width>
      <height>60</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Gym Management</string>
     </property>
    </widget>
    <widget class="QLabel" name="current_user">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>10</y>
       <width>150</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 12pt;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/rsc/img/user (1).png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="sign_out">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>0</y>
       <width>70</width>
       <height>60</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>sign out</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background-color: transparent;
padding: 0px; 
margin: 0px; </string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="login_as">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>40</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 9pt;



</string>
     </property>
     <property name="text">
      <string>Login Profile</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>1110</width>
      <height>660</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>page1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>page2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>page3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>page4</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>page5</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
